src/redisraft.h

```diff
@@ -859,6 +881,9 @@ int fsyncFile(int fd);
 int fsyncFileAt(const char *path);
 void fsyncDir(const char *path);
 int syncRename(const char *oldname, const char *newname);
+RaftRedisCommandArray *RaftRedisBlockToNonBlockCmds(RaftRedisCommandArray *cmds);
+RedisModuleString **RaftRedisExtractBlockedKeys(RaftRedisCommandArray *cmds, size_t *count);
```

Comment:
```suggestion
RaftRedisCommandArray *RaftRedisBlockToNonBlockCmds(RaftRedisCommandArray *cmds);
RedisModuleString **RaftRedisExtractBlockedKeys(RaftRedisCommandArray *cmds, size_t *count);
```
seems like these are leftovers

Reply:
must have removed already